src/main/java/core/basesyntax/List.java

```diff
@@ -17,5 +17,7 @@
 
     int size();
 
+    T[] toArray();
```

Comment:
I'm not sure if it was necessary to make changes to this class, maybe it's better to make the implementation in ArrayList